src/penumbra.worker.js

```diff
@@ -49,8 +49,13 @@ class PenumbraWorker {
           'PenumbraDecryptionWorker.get(): RemoteResource missing URL',
         );
       }
+      const {
+        requestInit = {
+          credentials: 'include',
```

Comment:
i think default should be undefined, not credentials include

Reply:
I can invert the default usage, though I feel that most users will be using this with our privacy center or with a URL that needs credentials.

Reply:
ya not so sure. i wasn't using a protected route before, was instead including a token in the URL that was used as auth. as soon as you include credentials, it actually had cors related errors since you cannot use a wildcard when include: credentials is true